fairlearn/preprocessing/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+"""This module contains algorithms implementing preprocessing steps that might help
+filter information that correlates with sensitive attributes.
+"""
+
+from ._informationfilter import InformationFilter
```

Comment:
so far we've used the convention of separating words in file names with underscores, i.e., _information_filter.py

Any particular reason not to do that?

Reply:
Did not realise that. But can change for sure.